src/cts/include/cts/TritonCTS.h

```diff
@@ -99,8 +100,11 @@ class TritonCTS
   int setClockNets(const char* names);
   void setBufferList(const char* buffers);
   void inferBufferList(std::vector<std::string>& bufferVector);
+  bool isClockBufferCandidate(sta::LibertyCell* buffer);
   void setRootBuffer(const char* buffers);
   std::string selectRootBuffer(std::vector<std::string>& bufferVector);
+  void setSinkBuffer(const char* buffers);
+  std::string selectSinkBuffer(std::vector<std::string>& bufferVector);
```

Comment:
In general it is best to do things like string argument parsing in tcl and take a list of db objects here, eg
```
odb::dbMaster* selectSinkBuffer(std::vector<odb::dbMaster*>& buffers);
```
Also you don't need to put Vector in the name when its clearly part of the type.

Reply:
OK.  I'll try to pass db objects from Tcl next time.  I'm just modifying legacy codes here.
bufferVector has been renamed as buffers